Sources/MountebankSwift/Api/MountebankValidationError.swift

```diff
@@ -2,13 +2,29 @@ import Foundation
 
 /// Error thrown if something did go wrong with the Mountebank client or server.
 public enum MountebankValidationError: Error, Equatable {
-
     /// The request to make is invalid
     case invalidRequestData
 
     /// The response from the Mountebank server is invalid
     case invalidResponseData
 
+    /// The response from the Mountebank server is invalid
+    case jsonDecodeError(Error)
```

Comment:
Perhaps use an equatable error type here to get rid of the == below?

Reply:
Is that possible with the errors thrown of `decodable` en `encodable`?

Reply:
reverted